core/primer.md

```diff
@@ -2,6 +2,185 @@
 
 <!-- no verify-specs -->
 
+## Abstract
+
+This non-normative document provides an overview of the xRegistry specification.
+It is meant to complement the xRegistry specification to provide additional
+background and insight into the history and design decisions made during the
+development of the specification. This allows the specification itself to focus
+on the normative technical details.
+
+## Table of Contents
+
+- [History](#history)
+- [Motivation](#motivation)
+- [Design Goals](#design-goals)
+- [Representations](#representations)
+- [Possible Use Cases](#possible-use-cases)
+- [Design decisions or topics of
+  interest](#design-decisions-or-topics-of-interest)
+
+## History
+
+The CNCF Serverless Working group was originally created by the CNCF's Technical
+Oversight Committee to investigate Serverless Technology and to recommend some
+possible next steps for some CNCF related activities in this space. After
+creating the common event format [CloudEvents](https://github.com/cloudevents)
+as a foundation for an interoperable ecosystem for event-driven applications the
+next step was to create a metadata model for declaring CloudEvents, their
+payloads and associating those declarations with application endpoints. As a
+result, the xRegistry (extensible registry) specification was created.
+
+xRegistry was initially part of CloudEvents, called "CloudEvents Discovery" but
+later moved into it's own repository and CNCF project.
```

Comment:
I'd drop "CNCF" for now. While yes it's a project under the CNCF, phrasing it this way makes it sound like an incubator project, or something more official. As of now it's more like a side-project and at some point we'll need to bring it up for formal status within the CNCF.